test/instrumentations/next/next.test.ts

```diff
@@ -0,0 +1,101 @@
+import { execSync } from 'child_process';
+import { versionsToTest } from '../../utils/versions';
+import { TestApp } from '../../utils/test-apps';
+import { installPackage, reinstallPackages, uninstallPackage } from '../../utils/test-setup';
+import { mkdirSync } from 'fs';
+import { join } from 'path';
+import { itTest } from '../../integration/setup';
+import { getSpanByName } from '../../utils/spans';
+import { SpanKind } from '@opentelemetry/api';
+const net = require('net');
+
+
+async function getPortFree() {
+    return new Promise( res => {
+        const srv = net.createServer();
+        srv.listen(0, () => {
+            const port = srv.address().port
+            srv.close((err) => res(port))
+        });
+    })
+}
```

Comment:
IIRC this exists in another file - let's extract it to a shared util function please

Reply:
ok,